test/e2e/request.test.js

```diff
@@ -0,0 +1,33 @@
+const Request = require('../../lib/api/request')
+const expect = require('chai').expect
+
+describe('Amio API Request', function () {
+  describe('create headers', () => {
+    it('return empty object - empty arguments', async () => {
+      function requestHeaders() {
```

Comment:
DRY: vyndej `requresHeaders` mimo `describe`.

Reply:
No jasne... copy paste bez uklidu. :-1: Aby to vypadalo ze je toho vic. :-D